### PR TITLE
Fix structure of HTMLMenuElement

### PR DESCRIPTION
**Pull request by teoli2003**
I'm preparing this area for a call to action for community members.

It is easier to fix the interface than to describe how to fix it.

### CODE REVIEW

**Review by skyclouds2001 — COMMENTED** [review]
I think the *Constructor* section can be removed, because `HTMLMenuElement` is not constructible

**Comment by teoli2003** [comment]
Strictly speaking it is in the spec, but as no browser implement it for the moment, let's get rid of it.

File: files/en-us/web/api/htmlmenuelement/index.md [hunk 1]
```diff
@@ -7,30 +7,16 @@ browser-compat: api.HTMLMenuElement
 
 {{APIRef("HTML DOM")}}
 
-The **`HTMLMenuElement`** interface provides special properties (beyond those defined on the regular {{DOMxRef("HTMLElement")}} interface it also has available to it by inheritance) for manipulating {{HTMLElement("menu")}} elements.
+The **`HTMLMenuElement`** interface provides additional properties (beyond those inherited form the {{domxref("HTMLElement")}} interface ) for manipulating a {{HTMLElement("menu")}} element, a semantic alternative to a {{HTMLElement("ul")}} element.
```
Comment (hamishwillee) [inline files/en-us/web/api/htmlmenuelement/index.md:10]: 2. Unnecessary whitespace
```suggestion
The **`HTMLMenuElement`** interface provides additional properties (beyond those inherited form the {{domxref("HTMLElement")}} interface) for manipulating a {{HTMLElement("menu")}} element, a semantic alternative to a {{HTMLElement("ul")}} element.
```

File: files/en-us/web/api/htmlmenuelement/index.md [hunk 2]
```diff
@@ -7,30 +7,16 @@ browser-compat: api.HTMLMenuElement
 
 {{APIRef("HTML DOM")}}
 
-The **`HTMLMenuElement`** interface provides special properties (beyond those defined on the regular {{DOMxRef("HTMLElement")}} interface it also has available to it by inheritance) for manipulating {{HTMLElement("menu")}} elements.
+The **`HTMLMenuElement`** interface provides additional properties (beyond those inherited form the {{domxref("HTMLElement")}} interface) for manipulating a {{HTMLElement("menu")}} element, a semantic alternative to a {{HTMLElement("ul")}} element.
```
Comment (hamishwillee) [inline files/en-us/web/api/htmlmenuelement/index.md:10]: 1. Normally we do "interface of the XXxxxx API?
2. Perhaps split this into sentences?

```suggestion
The **`HTMLMenuElement`** interface provides additional properties (beyond those inherited form the {{domxref("HTMLElement")}} interface) for manipulating a {{HTMLElement("menu")}} element.
`<menu>` is a semantic alternative to the {{HTMLElement("ul")}} element.
```

Comment (teoli2003) [reply files/en-us/web/api/htmlmenuelement/index.md]: 1. We don't really do the _of the xxx API_ for the HTML DOM, isn't it? (No strong feelings, but it sounds "heavy" to me)
2. Yes for two.

File: files/en-us/web/api/htmlmenuelement/index.md [hunk 3]
```diff
@@ -7,30 +7,17 @@ browser-compat: api.HTMLMenuElement
 
 {{APIRef("HTML DOM")}}
 
-The **`HTMLMenuElement`** interface provides special properties (beyond those defined on the regular {{DOMxRef("HTMLElement")}} interface it also has available to it by inheritance) for manipulating {{HTMLElement("menu")}} elements.
+The **`HTMLMenuElement`** interface provides additional properties (beyond those inherited form the {{domxref("HTMLElement")}} interface) for manipulating a {{HTMLElement("menu")}} element.
```
Comment (dawei-wang) [inline files/en-us/web/api/htmlmenuelement/index.md:10]:
```suggestion
The **`HTMLMenuElement`** interface provides additional properties (beyond those inherited from the {{domxref("HTMLElement")}} interface) for manipulating a {{HTMLElement("menu")}} element.
```

**Review by hamishwillee — APPROVED** [review]
Works for me. Approving.
